tests/unit/containers/admin-portal/AdminPortal.spec.jsx

```diff
@@ -0,0 +1,53 @@
+import { configureStore } from '@reduxjs/toolkit'
+import reducer from '~/redux/reducer'
+import {
+  createMemoryRouter,
+  createRoutesFromElements,
+  RouterProvider
+} from 'react-router-dom'
+import { routerConfig } from '~/router/router'
+import { render, screen } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import { ThemeProvider } from '@mui/material/styles'
+import { theme } from '~/styles/app-theme/custom-mui.styles'
+import { beforeEach, expect } from 'vitest'
+
+import AdminPortal from '~/containers/layout/admin-portal/AdminPortal'
+
+const renderAdminPortalRouter = (initialEntries, preloadedState) => {
+  const store = configureStore({
+    reducer: { appMain: reducer },
+    preloadedState
+  })
+  const router = createMemoryRouter(createRoutesFromElements(routerConfig), {
+    initialEntries: [initialEntries]
+  })
+
+  render(
```

Comment:
there is no need to add wrapping all these providers by yourself, we already have util [renderWithProviders](https://github.com/ita-social-projects/SpaceToStudy-Client/blob/82cefb038667db2ce8bfd941144bc5ce59e0ddab/tests/test-utils.jsx#L16) for that

Reply:
I have tried to use renderWithProviders and passed preloadedState with userRole - admin and initialEntries - '/admin', but in such case only empty div is rendered after AdminNavBar. Outlet component depends on routes and Suspence fallback is not working since Outlet is not loading. That is why I decided to add wrapping. 

<img width="445" alt="Знімок екрана 2023-10-05 о 15 23 36" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/6d879723-b988-4f2d-b3ad-02107116c209">

<img width="594" alt="Знімок екрана 2023-10-05 о 15 22 55" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/17857767/16d69a4c-6b7d-4877-ac21-0191764c1f85">